volume/drivers/adapter.go

```diff
@@ -16,7 +15,7 @@ var (
 
 type volumeDriverAdapter struct {
 	name         string
-	baseHostPath string
+	scopePath    func(s string) string
```

Comment:
Should this be `scopedPath` (with a `d`)?

Reply:
I did this as is because it's an action e.g. `scopPath("/foo")`, otherwise it would be `getScopedPath("/foo")`

Reply:
Oh! I see :blush: